google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -99,11 +116,6 @@ public OSProcessHandler createProcessHandler(
             executor.run();
 
             Process devappserverProcess = executor.getProcess();
-            if (devappserverProcess == null) {
```

Comment:
We shouldn't get rid of this block, because it is still possible that dev_appserver blows up for some reason and we don't want it to look nasty on the IDE.

How about we throw an ExecutionException with a generic message like "The Google App Engine local development server terminated with an error."?